cmd/app/main.go

```diff
@@ -32,7 +31,7 @@ func start(log *slog.Logger) error {
 	_ = env.Load()
 
 	// Catch signals to gracefully shut down the app
-	ctx, stop := signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
+	ctx, stop := signal.NotifyContext(context.Background(), os.Kill, os.Interrupt)
```

Comment:
AFAIK, `SIGKILL` can't really be caught, so it doesn't make sense to try to handle that gracefully?

Reply:
You are right. os.Interrupt should be sufficient here.

Reply:
Understandable. I think then the PR is not useful at all.